src/biz/tugay/groovyship/cli/BoardCommandLinePrinter.groovy

```diff
@@ -0,0 +1,45 @@
+package biz.tugay.groovyship.cli
+
+
+import biz.tugay.groovyship.modal.Board
+import biz.tugay.groovyship.modal.Coordinate
+import biz.tugay.groovyship.modal.Ship
+import biz.tugay.groovyship.service.ShipService
+
+class BoardCommandLinePrinter
+{
+  ShipService shipService = new ShipService()
+
+  void print(Board board) {
+    def noAttempt = ' '
+    def missedAttempt = '-'
+    def successfulAttempt = '□'
+    def sankShip = '◼'
+
+    print '   '
+    for (column in 0..<board.boardSize) {
+      print "$column  "
```

Comment:
These are called GStrings. The learn more about them see: https://groovy-lang.org/syntax.html#all-strings